src/Files.Uwp/Views/SettingsPages/Preferences.xaml

```diff
@@ -1,455 +1,477 @@
+                        </local:SettingsBlockControl>
+                        <local:SettingsBlockControl Title="{helpers:ResourceString Name=DisplayDateColumn}" HorizontalAlignment="Stretch">
+                            <local:SettingsBlockControl.Icon>
+                                <FontIcon Glyph="&#xE7B3;" />
+                            </local:SettingsBlockControl.Icon>
+                            <ToggleSwitch
+                                AutomationProperties.Name="{helpers:ResourceString Name=DisplayDateColumn}"
+                                IsOn="{Binding ShowDateColumn, Mode=TwoWay}"
+                                Style="{StaticResource RightAlignedToggleSwitchStyle}" />
+                        </local:SettingsBlockControl>
+                        <local:SettingsBlockControl Title="{helpers:ResourceString Name=DisplayDateCreatedColumn}" HorizontalAlignment="Stretch">
+                            <local:SettingsBlockControl.Icon>
+                                <FontIcon Glyph="&#xE7B3;" />
+                            </local:SettingsBlockControl.Icon>
+                            <ToggleSwitch
+                                AutomationProperties.Name="{helpers:ResourceString Name=DisplayDateCreatedColumn}"
+                                IsOn="{Binding ShowDateCreatedColumn, Mode=TwoWay}"
+                                Style="{StaticResource RightAlignedToggleSwitchStyle}" />
+                        </local:SettingsBlockControl>
+
+                        <local:SettingsBlockControl Title="{helpers:ResourceString Name=ResetLayoutPreferencesForAllFolders}" HorizontalAlignment="Stretch">
+                            <local:SettingsBlockControl.Icon>
+                                <FontIcon Glyph="&#xE777;" />
+                            </local:SettingsBlockControl.Icon>
+                            <StackPanel
+                                VerticalAlignment="Center"
+                                Orientation="Horizontal"
+                                Spacing="12">
+                                <FontIcon
+                                    Width="16"
+                                    VerticalAlignment="Center"
+                                    FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                    FontSize="16"
+                                    Glyph="&#xE001;"
+                                    Visibility="{Binding IsLayoutResetCheckmarkVisible}" />
+                                <Button
+                                    AutomationProperties.Name="{helpers:ResourceString Name=ResetLayoutPreferences}"
+                                    Command="{Binding ResetLayoutPreferencesCommand}"
+                                    Content="{helpers:ResourceString Name=ResetLayoutPreferences}" />
+                            </StackPanel>
+                        </local:SettingsBlockControl>
```

Comment:
What do you think about adding this right after the "force layout" option on line 279?

Reply:
The description for that setting says "these options", as in "the following options". I think I'd be strange to see the reset button among those.
Plus reset buttons are usually at the bottom.

Reply:
Perhaps it can be on the header of the expander? Might be more intuitive over there.

Reply:
![image](https://user-images.githubusercontent.com/39923744/185815658-8af27cce-a752-4c93-b6e3-eed998fa8960.png)

Reply:
Why not, the header needs to be changed to just "Folder preferences" though.

Reply:
But the it's a bit to easy to press by mistake. If put there we also need to add a confirmation step.

Reply:
It would still be a separate button so I think that would prevent accidental presses, kind of like the option to copy the file version.

Reply:
We could do like github and force users to type something to confirm reset, like:
`Type "files_is_best" to reset layout pref`

![tumblr_mhm353i5Fv1qea4hso1_640.png](https://user-images.githubusercontent.com/9673091/185816268-6964f43b-ced8-4fe8-b778-b375e2367d50.png)

Reply:
I actually find those prompts a little annoying 😂 
Either way, that sort of prompt or just a teaching tip with an additional confirm button should do the trick.